gulpfile.js

```diff
@@ -1056,44 +1156,50 @@ function checkTypes() {
     return './extensions/' + extension.name + '/' +
         extension.version + '/' + extension.name + '.js';
   }).sort();
-  return compileCss().then(() => {
-    return Promise.all([
-      closureCompile(compileSrcs.concat(extensionSrcs), './dist',
-          'check-types.js', {
-            include3pDirectories: true,
-            includePolyfills: true,
-            extraGlobs: ['src/inabox/*.js'],
-            typeCheckOnly: true,
-          }),
-      // Type check 3p/ads code.
-      closureCompile(['./3p/integration.js'], './dist',
-          'integration-check-types.js', {
-            externs: ['ads/ads.extern.js'],
-            include3pDirectories: true,
-            includePolyfills: true,
-            typeCheckOnly: true,
-          }),
-      closureCompile(['./3p/ampcontext-lib.js'], './dist',
-          'ampcontext-check-types.js', {
-            externs: ['ads/ads.extern.js'],
-            include3pDirectories: true,
-            includePolyfills: true,
-            typeCheckOnly: true,
-          }),
-      closureCompile(['./3p/iframe-transport-client-lib.js'], './dist',
-          'iframe-transport-client-check-types.js', {
-            externs: ['ads/ads.extern.js'],
-            include3pDirectories: true,
-            includePolyfills: true,
-            typeCheckOnly: true,
-          }),
-    ]);
-  }).then(() => {
-    if (isTravisBuild()) {
-      // New line after all the compilation progress dots on Travis.
-      console.log('\n');
-    }
-  }).then(() => exitCtrlcHandler(handlerProcess));
+  return compileCss()
+      .then(async() => {
+        await startNailgunServer();
```

Comment:
Do concurrent with compileCss? Start first and only await returned promise here?

Reply:
Tried this yesterday. `compileCss` takes a few milliseconds to run, so this makes no difference to speed, while being harder to maintain.